packages/app-store/salesforce/components/EventTypeAppCardInterface.tsx

```diff
@@ -60,49 +57,6 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
     recordOptions.find((option) => option.value === createEventOn) ?? recordOptions[0]
   );
 
-  const fieldTypeOptions = [
```

Comment:
Since we abstracted this componenet we don't need these options in the main Salesforce component

![image](https://github.com/user-attachments/assets/b44955af-fc6a-4f80-80b6-3e7c6b6e8122)